pkg/driverbuilder/templates.go

```diff
@@ -90,7 +93,101 @@ func LoadMakefileObjList(c builder.Config) (string, error) {
 			return strings.Split(l, "@PROBE_NAME@-y += ")[1], nil
 		}
 	}
-	return "", fmt.Errorf("obj list not found")
+	return "", fmt.Errorf("kmod obj list not found")
+}
+
+type makefileBpfData struct {
+	MakeObjList string
+}
+
+const makefileBpfTemplate = `
```

Comment:
Biggest change in the diff: added the current libs master bpf Makefile as a template.